.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Question
+about: Ask any questions about the project
+
+---
+
+/triage support
+
+**Question:**
+[Ask any question about this project but please include relevant details such as versions, screenshots, or error output as well as what you've tried already.]
```

Comment:
We don't want to encourage people asking support questions in the issues due to the poor visibility issues have. We should promote stackoverflow and slack channels in our documentation (README/website/releases) for support questions

Reply:
Removed.